CHANGELOG.md

```diff
@@ -5,6 +5,190 @@ information on the list of deprecated flags and APIs please have a look at
++ Allow client to talk to an older server [#27745](https://github.com/docker/docker/pull/27745)
+* Inform user client-side that a container removal is in progress [#26074](https://github.com/docker/docker/pull/26074)
++ Add `Isolation` to the /info endpoint [#26255](https://github.com/docker/docker/pull/26255)
++ Add `userns` to the /info endpoint [#27840](https://github.com/docker/docker/pull/27840)
+- Do not allow more than one mode be requested at once in the services endpoint [#26643](https://github.com/docker/docker/pull/26643)
+
+### Runtime
+
++ Add `--experimental` daemon flag to enable experimental features, instead of shipping them in a separate build [#27223](https://github.com/docker/docker/pull/27223)
++ Add a `--shutdown-timeout` daemon flag to specify the default timeout (in seconds) to stop containers gracefully before daemon exit [#23036](https://github.com/docker/docker/pull/23036)
++ Add `--stop-timeout` to specify the timeout value (in seconds) for individual containers to stop [#22566](https://github.com/docker/docker/pull/22566)
++ Add a new daemon flag `--userland-proxy-path` to allow configuring the userland proxy instead of using the hardcoded `docker-proxy` from `$PATH` [#26882](https://github.com/docker/docker/pull/26882)
++ Add a new daemon flag `--init-path` to allow configuring the path to the `docker-init` binary [#26941](https://github.com/docker/docker/pull/26941)
++ Add support for live reloading insecure registry in configuration [#22337](https://github.com/docker/docker/pull/22337)
++ Add support for storage-opt size on Windows daemons [#23391](https://github.com/docker/docker/pull/23391)
+* Improve reliability of `docker run --rm` by moving it from the client to the daemon  [#20848](https://github.com/docker/docker/pull/20848)
++ Add support for `--cpu-rt-period` and `--cpu-rt-runtime` flags, allowing containers to run real-time threads when `CONFIG_RT_GROUP_SCHED` is enabled in the kernel [#23430](https://github.com/docker/docker/pull/23430)
+* Allow parallel stop, pause, unpause [#24761](https://github.com/docker/docker/pull/24761)
+* Implement XFS quota for overlay2 [#24771](https://github.com/docker/docker/pull/24771)
+- Fix partial/full filter issue in `service tasks --filter` [#24850](https://github.com/docker/docker/pull/24850)
+- Allow engine to run inside a user namespace [#25672](https://github.com/docker/docker/pull/25672)
+- Fix a race condition between device deferred removal and resume device, when using the devicemapper graphdriver [#23497](https://github.com/docker/docker/pull/23497)
+- Add `docker stats` support in Windows [#25737](https://github.com/docker/docker/pull/25737)
+- Allow using `--pid=host` and `--net=host` when `--userns=host` [#25771](https://github.com/docker/docker/pull/25771)
++ Add metrics output [#25820](https://github.com/docker/docker/pull/25820)
```

Comment:
Just realized this is "experimental", so we should prefix with `(experimental)` ?